docs/docs/user/projects/models/update.md

```diff
@@ -3,22 +3,28 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-# Update a Model to a Newer Version
+# Update a Capella Model to a Newer Version
 
 <!-- prettier-ignore -->
 !!! warning
     Create backups (e.g. in a Git repository) before you start the update process.
-    A downgrade of models may not be possible (depends on the tool).
+    A downgrade of Capella models is not possible.
 
 <!-- prettier-ignore -->
 !!! info
-    If you want to update a Capella model, please follow the Capella specific instructions instead:
+    If you want to update a Capella model saved in TeamForCapella, please follow the Capella specific instructions instead:
     [Update a TeamForCapella based Capella model](../../tools/capella/teamforcapella/update.md)
 
-1. Open a new persistent session with the old version of your tool.
-1. Load the model with the old version into your persistent workspace.
-1. Close the session.
-1. Remove the automatic backup option in the Capella Collaboration Manager,
-   we'll rewire it at the end of this tutorial.
+1. Open a new persistent session with the old Capella version.
+1. Import the model into your workspace.
+1. Terminate the session.
+1. Update the model version to the newer Capella version:
+   [Change model metadata](metadata.md)
 1. Create a new session with the new Capella version.
-1. Migrate the model in your local workspace.
+1. In your persistent workspace loaded, execute the following steps:
+   <!-- prettier-ignore -->
+    1. Right click on the model in your workspace.
+    1. Select `Migration` and `Migrate Project toward current version`
+       ![Migrate Capella model](../../tools/capella/teamforcapella/migrate-capella-model.png)
+    1. A backup is  required.
+       ![Disable backup option](../../tools/capella/teamforcapella/backup-migration.png)
```

Comment:
In the info-box, you wrote: "If you want to update a Capella model saved in TeamForCapella, please follow the Capella specific instructions instead".

This is why I assume that this information is just about local Capella models in the workspace. But in these steps, you link to the TeamForCapella guide.

Reply:
so shall I remove the link to the backups?

Reply:
it is moved to `tools > capella` as it was about updating models and is now capella specific
so it is the file on which you commented later on